package.json

```diff
@@ -14,5 +14,9 @@
     "@web/dev-server": "^0.1.28",
     "@web/dev-server-esbuild": "^0.2.16",
     "typescript": "^4.5.2"
+  },
+  "dependencies": {
+    "workbox-routing": "^6.4.1",
+    "workbox-strategies": "^6.4.1"
```

Comment:
We should probably think about whether we want to use workbox or not. I'm fine either way, but I don't think it gives us too much. (SWR is easy enough to implement, and we don't have sophisticated logic.)